src/RustyObjectStore.jl

```diff
@@ -1,11 +1,17 @@
 module RustyObjectStore
 
-export init_object_store, blob_get!, blob_put, AzureCredentials, ObjectStoreConfig
```

Comment:
Are we purposefully not exporting `StaticConfig` or was that just an oversight?

Reply:
Oversight. Will fix!